CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributor Onboarding
+Thank you for taking the time to contribute to Azure Functions in [Java](https://go.java/)
+
+## Table of Contents
+
+- [What should I know before I get started?](#what-should-i-know-before-i-get-started)
+- [Pre-requisites](#pre-requisites)
+- [Pull Request Change Flow](#pull-request-change-flow)
+- [Development Setup](#development-setup)
+- [Pre Commit Tasks](#pre-commit-tasks)
+- [Continuous Integration Guidelines & Conventions](#continuous-integration-guidelines-&-conventions)
+- [Getting Help](#getting-help)
+
+## What should I know before I get started
+- [Azure Functions Java Overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function-azure-cli?tabs=bash%2Cbrowser&pivots=programming-language-java)
+- [Azure Function Java developer guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-java?tabs=consumption)
+
+## Pre-requisites
+
+- OS
+    - MacOS, Ubuntu (or) Windows10
+- Language Runtimes
+    - .NET Core 2.0
+    -  Java 1.8
+
+- IDE
+   - [IntelliJ](https://www.jetbrains.com/idea/download)
+   - [Java extension in VS Code](https://code.visualstudio.com/docs/languages/java)
+- Java Tools 
+    - [Maven](https://maven.apache.org/install.html)
+    - [JDK (openJDK)  8 and/or 11](https://openjdk.java.net/install/)
```

Comment:
We use Zulu OpenJDK. https://docs.microsoft.com/en-us/azure/developer/java/fundamentals/java-jdk-long-term-support